src/pages/home/report/ReportActionItem.js

```diff
@@ -27,9 +27,19 @@ const propTypes = {
     // Should the comment have the appearance of being grouped with the previous comment?
     displayAsGroup: PropTypes.bool.isRequired,
 
+    // Is this the most recent IOU Action?
+    isMostRecentIOUReport: PropTypes.bool.isRequired,
+
     /* --- Onyx Props --- */
     // List of betas for the current user.
     betas: PropTypes.arrayOf(PropTypes.string),
+
+    // The report currently being looked at
+    report: PropTypes.shape({
+
+        // IOU report ID associated with current report
+        iouReportID: PropTypes.number,
```

Comment:
Any reason not to just pass the reportID at this stage, and subscribe to the full report Onyx further down the Component chain?

Reply:
Sure, I can pass just iouReportID here. 👍

Reply:
I'm not sure this was addressed and I think @Julesssss was suggesting that we pass the `iouReportID` directly instead of subscribing to the entire report object.